src/Game.cpp

```diff
@@ -179,6 +179,15 @@ bool Game::ProcessEvents(const SDL_Event& event)
 		{
 			return false;
 		}
+		else if (event.window.event == SDL_WINDOWEVENT_SIZE_CHANGED)
+		{
+			int width, height;
+			_window->GetSize(width, height);
+			auto aspect = _window->GetAspectRatio();
+			
+			bgfx::reset(width, height, BGFX_RESET_NONE);
+			_camera->SetProjectionMatrixPerspective(70.0f, aspect, 1.0f, 65536.0f);
```

Comment:
These magic numbers should be set as named constants and eventually should come from game data or config file.
I would recommend you make an entry in the Config struct and use it in all the places we set FOV and near and far.

Reply:
I added three new parameters in Config struct. These are used at least at two point, at camera initialization, and on resize.
I used your static_cast presentation.